src/atoms/RadioButton/Backdrop.native.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components/native';
+import View from '../View';
+
+const Backdrop = styled(View)(
+  (props) =>
+    `
+    width: ${props.width};
+    height: ${props.height};
```

Comment:
Use https://github.com/razorpay/blade/pull/51

Reply:
Not using here as discussed!